test/parallel/test-cluster-net-send.js

```diff
@@ -1,55 +1,56 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var fork = require('child_process').fork;
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+const net = require('net');
 
 if (process.argv[2] !== 'child') {
   console.error('[%d] master', process.pid);
 
-  var worker = fork(__filename, ['child']);
-  var called = false;
+  const worker = fork(__filename, ['child']);
+  let called = false;
 
-  worker.once('message', function(msg, handle) {
-    assert.equal(msg, 'handle');
+  worker.once('message', common.mustCall((msg, handle) => {
+    assert.strictEqual(msg, 'handle');
     assert.ok(handle);
     worker.send('got');
 
-    handle.on('data', function(data) {
+    handle.on('data', common.mustCall((data) => {
       called = true;
-      assert.equal(data.toString(), 'hello');
-    });
+      assert.strictEqual(data.toString(), 'hello');
+    }));
 
-    handle.on('end', function() {
+    handle.on('end', () => {
       worker.kill();
     });
-  });
+  }));
 
-  process.once('exit', function() {
+  process.once('exit', () => {
     assert.ok(called);
   });
 } else {
   console.error('[%d] worker', process.pid);
 
-  var socket;
-  var cbcalls = 0;
+  let socket;
+  let cbcalls = 0;
   function socketConnected() {
     if (++cbcalls === 2)
       process.send('handle', socket);
   }
 
-  var server = net.createServer(function(c) {
-    process.once('message', function(msg) {
-      assert.equal(msg, 'got');
+  const server = net.createServer((c) => {
+    process.once('message', common.mustCall((msg) => {
+      assert.strictEqual(msg, 'got');
       c.end('hello');
-    });
+    }));
     socketConnected();
   });
-  server.listen(common.PORT, function() {
-    socket = net.connect(common.PORT, '127.0.0.1', socketConnected);
+  server.listen(0, () => {
+    socket = net.connect(server.address().port, server.address().address,
+      socketConnected);
   });
 
-  process.on('disconnect', function() {
+  process.on('disconnect', () => {
     process.exit();
     server.close();
```

Comment:
Isn't this unreachable code?

It would be better to have the process exit naturally, rather than call `process.exit()`.